source/content/guides/drupal/drupal-hosted-createbt/07-core.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Migrate a Site That Was Created with Build Tools to Drupal 9
+title: Migrate a Site That Was Created with Build Tools to Drupal latest
 subtitle: Update Drupal Core Using Composer
 description: 
-cms: "Drupal 9"
+cms: "Drupal"
```

Comment:
@stovak references `d9` in code samples.  see preview https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createbt/core